packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -75,6 +77,28 @@ export function getReadableSpanTransformer(
   };
 }
 
+function transformStatus(status: ot.Status): Status | undefined {
+  switch (status.code) {
+    case ot.StatusCode.UNSET:
+      return undefined;
+    case ot.StatusCode.OK:
+      return {code: Code.OK};
+    case ot.StatusCode.ERROR:
+      return {code: Code.UNKNOWN, message: status.message};
+    default: {
+      exhaust(status.code);
+      // TODO: log failed mapping
+      return {code: Code.UNKNOWN, message: status.message};
+    }
+  }
+}
+
+/**
+ * Assert switch case is exhaustive
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I'm surprised that this suppression is necessary - I guess it's because TS thinks that line 89 is unreachable?

Reply:
I think the lint is really just local to this function, the `switchValue` param isn't used in the body so it complains. I could also just return the value to avoid disabling lint